app/controllers/application_controller.rb

```diff
@@ -80,7 +80,7 @@ def require_user
       unless current_user
         store_location
         flash[:notice] = "You must be logged in to access this page"
-        redirect_to login_url+'?return_to=' + URI.encode(request.env['PATH_INFO'])
```

Comment:
Yeah this is a issue I should have handled separately raising a github issue.Will do it as per from the next time. Thanks Jeff!